<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>process</class>
 <widget class="QDialog" name="process">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>184</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>184</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>184</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LMOSL</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_11">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <widget class="QFrame" name="form_area">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="layout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layout_2">
          <item>
           <widget class="QLabel" name="ligth_source_label">
            <property name="text">
             <string>Ligth Source</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ligth_source">
            <property name="minimumSize">
             <size>
              <width>58</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>58</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AUX</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_3">
          <item>
           <widget class="QLabel" name="channels_label">
            <property name="text">
             <string>Channels</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="channels">
            <property name="minimumSize">
             <size>
              <width>55</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>512</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_4">
          <item>
           <widget class="QLabel" name="time_label">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QDoubleSpinBox" name="time">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="time_measurement">
              <property name="minimumSize">
               <size>
                <width>48</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>48</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>ms</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>s</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>us</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_6">
          <item>
           <widget class="QLabel" name="time_per_channel_label">
            <property name="text">
             <string>Time per Channel:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="time_per_channel">
            <property name="text">
             <string>0 ms</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontal_spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_7">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layout_8">
          <item>
           <widget class="QLabel" name="start_optical_power_label">
            <property name="text">
             <string>Start Optical Power (%)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="start_optical_power">
            <property name="minimumSize">
             <size>
              <width>55</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_9">
          <item>
           <widget class="QLabel" name="end_optical_power_label">
            <property name="text">
             <string>End Optical Power (%)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="end_optical_power">
            <property name="minimumSize">
             <size>
              <width>55</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontal_spacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_10">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layout_11">
          <item>
           <widget class="QLabel" name="final_temperature_label">
            <property name="text">
             <string>Final Temperature (°C)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="final_temperature">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <double>600.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_12">
          <item>
           <widget class="QLabel" name="heating_rate_label">
            <property name="text">
             <string>Heating Rate (°C/s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="heating_rate">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>20.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_13">
          <item>
           <widget class="QLabel" name="stabilization_label">
            <property name="text">
             <string>Stabilization (s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="stabilization">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontal_spacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttons_area">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="push_button_accept">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
        <property name="shortcut">
         <string notr="true">Return</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push_button_cancel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="shortcut">
         <string notr="true">Esc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push_button_information">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Information</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="vertical_spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
